### PR TITLE
chore: fix port, remove port on doc URLs

### PR DESCRIPTION
**Pull request by VinneyJ**
1. change port from 8082 to 80
2. changed domain from mediacloud.org to civicsignal.africa and removed 8082 port from in docs URLs,

Fixes #2

### CODE REVIEW

File: doc/api_2_0_spec/admin_api_2_0_spec.md [hunk 1]
```diff
@@ -163,11 +163,11 @@ Media Cloud API 2.0 Spec for a more detailed description of the `q` and `fq` par
 
 The output of these calls is in exactly the same format as for the api/v2/stories/single call.
 
-URL: https://api.mediacloud.org/api/v2/stories/list?last_processed_stories_id=8625915
+URL: https://aapp.civicsignal.africag/api/v2/stories/list?last_processed_stories_id=8625915
```
Comment (thepsalmist) [inline doc/api_2_0_spec/admin_api_2_0_spec.md]:
```suggestion
URL: https://app.civicsignal.africag/api/v2/stories/list?last_processed_stories_id=8625915
```

Comment (kilemensi) [inline doc/api_2_0_spec/admin_api_2_0_spec.md]:
```suggestion
URL: https://app.civicsignal.africag/api/v2/stories/list?last_processed_stories_id=8625915
```

File: doc/api_2_0_spec/api_2_0_spec.md [hunk 2]
```diff
@@ -1280,7 +1280,7 @@ Output:
 
 #### Example
 
-URL: <https://api.mediacloud.org/api/v2/auth/activate>
+URL: <https://app.mcivicsignal.africa/api/v2/auth/activate>
```
Comment (thepsalmist) [inline doc/api_2_0_spec/api_2_0_spec.md]:
```suggestion
URL: <https://app.civicsignal.africa/api/v2/auth/activate>
```

File: doc/api_2_0_spec/api_2_0_spec.md [hunk 3]
```diff
@@ -255,7 +255,7 @@ None.
 
 Fetching information on The New York Times
 
-URL: https://api.mediacloud.org/api/v2/media/single/1
+URL: https://app.civicsignal.africa:80/api/v2/media/single/1
```
Comment (thepsalmist) [inline doc/api_2_0_spec/api_2_0_spec.md]:
```suggestion
URL: https://app.civicsignal.africa/api/v2/media/single/1
```

File: doc/api_2_0_spec/api_2_0_spec.md [hunk 4]
```diff
@@ -1779,7 +1779,7 @@ Output:
 
 ### Example
 
-URL: https://api.mediacloud.org/api/v2/stats/list
+URL: https://app.mcivicsignal.africa/api/v2/stats/list
```
Comment (DavidTheProgrammer) [inline doc/api_2_0_spec/api_2_0_spec.md:1782]: https://app.civicsignal.africa/api/v2/stats/list

Comment (satoru260) [inline doc/api_2_0_spec/api_2_0_spec.md]:
```suggestion
URL: https://app.civicsignal.africa/api/v2/stats/list
```

**Review by DavidTheProgrammer — CHANGES_REQUESTED** [review]
LGTM aside from the typos

File: doc/api_2_0_spec/admin_api_2_0_spec.md [hunk 5]
```diff
@@ -452,7 +452,7 @@ for each tag_set included in the list of tags other than the tags added by this
 
 ### Example
 
-URL: https://api.mediacloud.org/aip/v2/stories/put_tags
+URL: https://app.civicsignal.africa/aip/v2/stories/put_tags
```
Comment (thepsalmist) [inline doc/api_2_0_spec/admin_api_2_0_spec.md]:
```suggestion
URL: https://app.civicsignal.africa/api/v2/stories/put_tags
```

**Review by thepsalmist — COMMENTED** [review]
LGTM

**Review by thepsalmist — APPROVED** [review]

File: doc/api_2_0_spec/admin_api_2_0_spec.md [hunk 6]
```diff
@@ -497,7 +497,7 @@ Output:
 
 ### Example
 
-https://api.mediacloud.org/api/v2/tags/create
+https:///api/v2/tags/create
```
Comment (kilemensi) [inline doc/api_2_0_spec/admin_api_2_0_spec.md]: Don't have full context but doesn't this need a hostname?

**Review by kilemensi — COMMENTED** [review]
👍🏽 .. don't have full context so if @thepsalmist is happy with it, please proceed to merge

---

1. Why is the **API** backend called `app`? Shouldn't it be `api.civicsignal.africa` instead of `app.civicsignal.africa`? `app.hostname` is usually a frontend/UI app e.g. https://app.gitbook.com, https://app.sendgrid.com/, etc.
2. Lets fix or remove the GitHub actions (Not in this PR)
Screenshot 2024-07-18 at 11 06 42

**Comment by thepsalmist** [comment]
> 👍🏽 .. don't have full context so if @thepsalmist is happy with it, please proceed to merge
> 
> 1. Why is the **API** backend called `app`? Shouldn't it be `api.civicsignal.africa` instead of `app.civicsignal.africa`? `app.hostname` is usually a frontend/UI app e.g. https://app.gitbook.com, https://app.sendgrid.com/, etc.
> 2. Lets fix or remove the GitHub actions (Not in this PR)
> 
> Screenshot 2024-07-18 at 11 06 42

I'm happy with making the domain changes to reflect this is an `api` and not `app` and make corresponding changes to the frontend applications

**Comment by VinneyJ** [comment]
Cool on it.

**Review by DavidTheProgrammer — APPROVED** [review]
LGTM

**Pull request merged**